cli.js

```diff
@@ -39,15 +41,52 @@ function toHex(number, length = 32) {
 }
 
 /** Display ETH account balance */
-async function printETHBalance({ address, name }) {
-  console.log(`${name} ETH balance is`, web3.utils.fromWei(await web3.eth.getBalance(address)))
+async function printETHBalance({ address, name, symbol }) {
+  console.log(`${name} balance is`, web3.utils.fromWei(await web3.eth.getBalance(address)),`${symbol}`)
 }
 
 /** Display ERC20 account balance */
 async function printERC20Balance({ address, name, tokenAddress }) {
   const erc20ContractJson = require('./build/contracts/ERC20Mock.json')
   erc20 = tokenAddress ? new web3.eth.Contract(erc20ContractJson.abi, tokenAddress) : erc20
-  console.log(`${name} Token Balance is`, web3.utils.fromWei(await erc20.methods.balanceOf(address).call()))
+  balance = await erc20.methods.balanceOf(address).call()
+  decimals = await erc20.methods.decimals().call()
+  console.log(`${name}`,(await erc20.methods.name().call()),`Token Balance is`,toDecimals(balance, decimals, (balance.length + decimals)).toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ","),(await erc20.methods.symbol().call()))
+}
+
+async function generateTransaction(contractAddress, gasLimit, encodedData, value = 0) {
+  const gasPrice = await fetchGasPrice()
+  let tx = {}
+  async function txoptions() {
```

Comment:
Instead of `contracts.method.call` function we are using `eth.accounts.signTransaction` to support EIP-1559 transactions.